articles/101_build_tool.md

```diff
@@ -199,8 +190,9 @@ The tool supports building CMake packages and builds them in isolation as well a
 
 `ament_tools` is provided by a standalone Python 3 package used to build ROS 2 packages.
 It was developed to bootstrap the ROS 2 project, is therefore only targeting Python 3, and works on Linux, MacOS and Windows.
-In addition to CMake packages it also supports building Python packages and can infer meta information without requiring an explicit package manifest.
+In addition to CMake packages it also supports building Python packages and can infer meta information without requiring an explicit package manifest (which is e.g. used for the FastRTPS package).
 The tool performs an "isolated" build like `catkin_make_isolated` and `catkin_tools` (one CMake invocation per package) and also parallelizes the build of packages which have no (recursive) dependencies on each other (like `catkin_tools`).
+While is coveres more build systems and platforms than `catkin_tools` it doesn't have any of `catkin_tools`s usability features like profiles, output handling, etc.
```

Comment:
not sure what you meant here, "while it covers" maybe?

Reply:
Fixed in 553d408a0762581565bc0aacd4e2032bbde864cd.